README.md

```diff
@@ -47,7 +47,7 @@ Go into `./examples/arduino-uno` (or the directory for whatever board you want),
 cd examples/arduino-uno
 
 # Build and run it on a connected board
-cargo run --bin uno-blink
+cargo run --release --bin uno-blink
```

Comment:
We really shouldn't require release builds.  If panic bloat is more of a problem now, we'll have to be more explicit about this topic or, worst-case, adjust the dev-profile even further.